src/queries/useAuthorityExport/useAuthorityExport.js

```diff
@@ -0,0 +1,44 @@
+import { useMutation, useQueryClient } from 'react-query';
+import { useOkapiKy, useNamespace } from '@folio/stripes/core';
+
+const useAuthorityExport = ({ onError, onSuccess, ...restOptions }) => {
+  const ky = useOkapiKy();
+  const queryClient = useQueryClient();
+  const [namespace] = useNamespace({ key: 'authorities' });
+
+  const customOptions = {
+    onError,
+    onSuccess: async (response) => {
+      // Creating a delay because result list takes some time to update
+      await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Also, no need to delay this code here. We do this when editing/deleting records because there's a delay on backend between a successful request and when search results are actually updated.
Because exporting doesn't affect the records we don't have to await here